main.py

```diff
@@ -8,11 +8,36 @@
 
 
 def set_arg_parser():
-    prog_desc = "Module de l'intelligence artificielle. L'option est de charger un fichier de configuration."
-    arg_parser = argparse.ArgumentParser(prog="RobocupULaval's Team ULtron AI", description=prog_desc)
-
-    arg_parser.add_argument('config_file', nargs='?', help="load a configuration file(.ini/cfg style)",
-                            default="config/sim.cfg")
+    prog_desc = 'Artificial intelligent and Engine software for the ULtron team in the RoboCup SSL.'
+    arg_parser = argparse.ArgumentParser(prog='ULtron\'s AI of the RoboCup ULaval group.', description=prog_desc)
+
+    arg_parser.add_argument('config_file',
+                            nargs='?',
+                            action='store',
+                            help='Load a configuration file(.ini/cfg style).',
+                            default='config/sim.cfg')
+
+    arg_parser.add_argument('--engine_fps',
+                            action='store',
+                            type=int,
+                            help='Set the engine FPS if engine fps is locked.',
+                            default=30)
+
+    arg_parser.add_argument('--unlock_engine_fps',
+                            action='store_true',
+                            help='Flag to unlock the engine FPS.',
+                            default=False)
+
+    arg_parser.add_argument('--on_negative_side',
+                            action='store_true',
+                            help='Flag when we are on the negative x side of the field.',
+                            default=False)
+    
+    arg_parser.add_argument('--queue_size',
+                            action='store',
+                            type=int,
+                            help='Max size of the inter process queues.',
+                            default=100)
```

Comment:
Is this parameter change often and should it be different on different computer?

Reply:
you can fill up some queues if your computer is really slow :P. We could always make them larger...

Reply:
Mode to the framework attribute for the moment.